lana/src/display/OpenFileInPackage.ts

```diff
@@ -8,22 +8,29 @@ import {
   Uri,
   ViewColumn,
   commands,
+  workspace,
   type TextDocumentShowOptions,
 } from 'vscode';
 
 import { Context } from '../Context.js';
 import { Item, Options, QuickPick } from './QuickPick.js';
 
+import { getMethodLine, parseApex } from '../salesforce/ApexParser/ApexSymbolLocator.js';
+
 export class OpenFileInPackage {
-  static async openFileForSymbol(
-    context: Context,
-    name: string,
-    lineNumber?: number,
-  ): Promise<void> {
-    const paths = await context.findSymbol(name);
+  static async openFileForSymbol(context: Context, symbolName: string): Promise<void> {
+    if (!symbolName?.length) {
+      return;
+    }
+
+    const parts = symbolName.split('.');
+    const fileName = parts[0];
+
+    const paths = await context.findSymbol(fileName as string);
```

Comment:
In context if the file could not be found we display the message `Type 'MyClass' was not found in workspace`.

I wonder if we should do similar if the symbol can not be found.
If the constructor or method has been removed (log was generated on old version) we could display
`Symbol 'myMethod' could not be found in file 'MyClass'`.

The file would still be opened though